october-2017/17.md

```diff
@@ -1,20 +1,57 @@
+* Modern Design
+  <span style="font-size: 14px; margin-bottom: 10px; display: block; opacity: .7;">We are making sure to get the site up to modern design standards.</span>
+* Fully Responsive
+  <span style="font-size: 14px; margin-bottom: 10px; display: block; opacity: .7;">This is part of a larger goal to get all MyBB products responsive, we're starting with the main site.</span>
+* SEO
+  <span style="font-size: 14px; margin-bottom: 10px; display: block; opacity: .7;">Finding pages on MyBB should be a much better experience through search engines.</span>
+
+If you have some time and want to leave some feedback, or if you just want to look at the new design, please visit the following thread. Thanks!
 
 [![New MyBB responsive design](../resources/visit-thread-white.svg)](https://community.mybb.com/thread-213603-post-1286609.html#pid1286609)
 
-##Blog
-In regards to the MyBB Blog we are hoping to make more use of this by providing project updates and make more of the things we are working on behind the scenes public! We have created a repo on GitHub which everyone can view and open issues. If you have ideas for Blog posts that you would like to share with us open an issue on Github and we will take a look at those. Although this is a strange thing to use Github for we plan to use labels (accepted, rejected, under discussion) and we are definitely open to suggestions regardless if it is a series of blog posts or just a single post. You can find the [Github repo for this here](https://github.com/mybb/blog.mybb.com-drafts).
+### Blog
+Our blog will be getting some love as well. We're now creating our blog posts via a repository on GitHub, so you can help! (Or just get a sneak peak at what the next blog will be about) If you have an idea for a blog post that we haven't done yet, or think we could be doing something better, feel free to open an issue in the repository linked below.
+
+[![New MyBB responsive design](../resources/github-transparent.svg)](https://github.com/mybb/blog.mybb.com-drafts)
+
+### Official Demo Site
+You may or may not know this, but we keep the community forum as vanilla as possible so users can get a feel for what MyBB is, right out of the box. Unfortunately this limits some decisions we make where improving the community forum is concerned. Not only that, but it limits how users can explore a vanilla MyBB installation as well. We are going to be creating a demo website to solve both of these issues, allowing you as users to browse a clean installation of MyBB easily, (It will be missing a couple of features, for security reasons) as well as letting us make some quality of life improvements to our community forum. You can expect work to begin on this project after the new website has launched. Stay tuned for further updates!
```

Comment:
`allowing you as users to browse a clean installation of MyBB easily,` - change the `,` to `.`.

> We are in the process of setting up a new demo system to solve both of these issues, allowing you as users to create and play with a clean installation of MyBB easily. Obviously these demo installs will be lacking a couple of features that a full blown install contains, such as the ability to install custom themes and plugins - this is purely a security precaution.
>
> Providing a clean demo system for users also opens up the possibility for more changes to be made to the official community forum - such as updating the styling to fit in with the rest of the website - in the future.
>
> You can expect work to begin on this project after the new website has launched. Stay tuned for further updates!